MaxText/layers/quantizations.py

```diff
@@ -362,6 +499,9 @@ def _get_quant_config(config):
     return "nanoo_fp8"
   if config.quantization == "aqt_fp8":
     return _get_aqt_fp8_quant_config(config)
+  if config.quantization == "aqt_fp8_full":
```

Comment:
Could we add a short comment on what is `aqt_fp8_full`? probably readers could get confused with `aqt_fp8` (like me, sorry).

Reply:
pls see the comment here: https://github.com/AI-Hypercomputer/maxtext/pull/1870/files#diff-17fed7d6a86f604c00d7bb8419e7469e9ef5c1d75c73aa48ee002af0410c3f2bR394